plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.ts

```diff
@@ -63,15 +64,15 @@ export const examples: TemplateExample[] = [
           input: {
             repoUrl:
               'bitbucket.org?repo=repo&workspace=workspace&project=project',
-            description: 'Initialize a git repository',
             repoVisibility: 'public',
           },
         },
       ],
     }),
   },
   {
-    description: 'Set the default branch.',
+    description:
+      'Initializes a Bitbucket repository with a default Branch, if not set defaults to master',
```

Comment:
```suggestion
      'Initializes a Bitbucket repository with a default branch, if not set defaults to main',
```

I can't get clear confirmation on this in the docs but generally it seems the default was changed to main at some point after 2020. I'm fine with being wrong on this, I always have a hard time finding things in the Atlassian docs.

Reply:
Same as my previous comment. DefaultBranch is set to master in the action

```
const {
        repoUrl,
        description,
        defaultBranch = 'master',
        repoVisibility = 'private',
        enableLFS = false,
        gitCommitMessage = 'initial commit',
        gitAuthorName,
        gitAuthorEmail,
      } = ctx.input;
```